examples/EnergyPlusAgent/README.md

```diff
@@ -0,0 +1,255 @@
+# EnergyPlus Agent
+
+This is an example agent that demonstrates how to integrate with EnergyPlus simulation, run building model simulations 
+with EnergyPlus, send/receive messages back and forth between VOLTTRON and
+EnergyPlus simulation. For more information about EnergyPlus, please refer to https://www.energyplus.net/sites/default/files/docs/site_v8.3.0/GettingStarted/GettingStarted/index.html  
+
+## EnergyPlus installation
+For installing setup in Ubuntu based systems, 
+
+1. Install java JDK package
+
+```
+sudo apt update
+sudo apt install default-jdk
+```
+
+2. Download and install EnergyPlus (ver 8.5.0)
+
+```
+wget https://github.com/NREL/EnergyPlus/releases/download/v8.5.0/EnergyPlus-8.5.0-c87e61b44b-Linux-x86_64.sh
+chmod +x EnergyPlus-8.5.0-c87e61b44b-Linux-x86_64.sh
+sudo ./EnergyPlus-8.5.0-c87e61b44b-Linux-x86_64.sh 
+```
+
+## EnergyPlus Agent Configuration
+
+1. Copy 'bcvtb' and 'eplus' folders into root of VOLTTRON source directories. 'eplus' should contain EnergyPlus Input Data File (IDF). IDF file is an ASCII file containing the data describing the building and HVAC system to be simulated.
```

Comment:
Where do I get these folders from?

Reply:
I have added link to 'eplus'. Waiting on Robert to give me link to 'bcvtb'.